libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/RoomInfo.kt

```diff
@@ -85,7 +85,7 @@ data class RoomInfo(
      */
     fun usersWithRole(role: RoomMember.Role): List<UserId> {
         return if (role is RoomMember.Role.Owner && role.isCreator) {
-            this.creators
+            if (privilegedCreatorRole) this.creators else emptyList()
```

Comment:
Maybe add a unit test to check that an empty list is returned when the conditions are met?